### PR TITLE
docs: Update Body class documentation

### PR DESCRIPTION
**Pull request by hbaklan943**
Added information about the setType and getType methods in the Body class documentation.

### CODE REVIEW

**Comment by shakiba** [comment]
This is where you need to remove @internal tag if you would like:
https://github.com/piqnt/planck.js/blob/master/src/dynamics/Body.ts#L388

**Comment by hbaklan943** [comment]
Sure, I am on it.

**Comment by shakiba** [comment]
Thanks!

Api docs also needs to be added as comments in the source code, otherwise they will be overwritten by next api doc generation.

If you like to see how it works you can run `npm run typedoc`. But please do not add generated docs in this PR, I will generate them a little bit later.

**Comment by hbaklan943** [comment]
Hmm, so I could have generated the docs using that command.

**Comment by hbaklan943** [comment]
I returned the docs to its original state and edited only the `src/dynamics/Body.ts` to add doc comment to method definitions, hopefully, that was what you wanted.

**Review by shakiba — APPROVED** [review]
Thanks!

File: docs/api/classes/body.md [hunk 1]
```diff
@@ -1106,4 +1106,4 @@ ___
 
 *Defined in [src/dynamics/Body.ts:586](https://github.com/shakiba/planck.js/blob/acc3bd8/src/dynamics/Body.ts#L586)*
 
-**Returns:** *void*
```
Comment (shakiba) [inline docs/api/classes/body.md:1109]: I guess this is change by mistake.

Comment (hbaklan943) [reply docs/api/classes/body.md:1109]: Yes, it was a mistake.

**Comment by shakiba** [comment]
> I reverted the docs back to its original state and edited only the `src/dynamics/Body.ts` to add doc comment to method definitions, hopefully, that was what you wanted.

Yes, looks great! I will re-generate api docs tomorrow. There has been many small changes in the api for the new version, so I need to review them before publishing api docs.

**Pull request merged**